src/cockpit/389-console/src/lib/ldap_editor/wizards/operations/addCosTemplate.jsx

```diff
@@ -0,0 +1,1237 @@
+                            <div>
+                                <Spinner className="ds-left-margin" size="md" />
+                                &nbsp;&nbsp;Adding entry ...
+                            </div>
+                        }
+                    </Alert>
+                </div>
+                {resultVariant === 'danger' &&
+                    <Card isSelectable>
+                        <CardTitle>LDIF Data</CardTitle>
+                        <CardBody>
+                            {ldifLines.map((line) => (
+                                <h6 key={line.id}>{line.data}</h6>
+                            ))}
+                        </CardBody>
+                    </Card>
+                }
+                <Modal
+                    variant={ModalVariant.small}
+                    title="Create CoS Template"
+                    isOpen={this.state.isConfirmModalOpen}
+                    onClose={this.handleConfirmModalToggle}
+                    actions={[
+                        <Button key="confirm" variant="primary" onClick={this.handleCreateTemplateEnd}>
+                        Create Template
+                        </Button>,
+                        <Button key="cancel" variant="link" onClick={this.handleConfirmModalToggle}>
+                        Cancel
+                        </Button>
+                    ]}
+                    >
+                    <Grid>
+                        <GridItem span={12}>
+                            <TextContent className="ds-margin-top">
+                                <Text>
+                                    You've chosen 'Classic' CoS type.
+                                    cosTemplateDN attribute is set to {this.props.wizardEntryDn}.
+                                </Text>
+                                <Text>
+                                    Do you want to create a CoS template now?
+                                </Text>
+                                <Text>
+                                    It will be added as a child to this entry: '{this.props.wizardEntryDn}'
+                                </Text>
+                            </TextContent>
+                        </GridItem>
+                    </Grid>
+                </Modal>
+            </div>
+        );
+
+        let finishButtonName;
+        if (this.props.cosDefCreateMoreTemplate) {
+            finishButtonName = 'Create More Templates';
+        } else if (this.props.definitionWizardEntryDn !== '') {
+            finishButtonName = 'Back to CoS Definition';
+        } else {
+            finishButtonName = 'Finish';
+        }
+
+        const addEntrySteps = [
+            {
+                id: 1,
+                name: 'Select Name & Priority',
+                component: namingValAndPriority,
+                enableNext: namingVal === '' ? false : true,
+                canJumpTo: stepIdReached >= 1 && stepIdReached < 7,
+                hideBackButton: true
+            },
```

Comment:
I think we need a step before this.  There is no way to move back to the main CoS landing page.  You can when entering definitions, but not when entering templates.  So its' not consistent, and if you click on the wrong item you have to completely exit and re-select "Class of Service" from the entry menu.

Reply:
I've just looked and it's the same like we have in 'New' section (with Group, Role, etc.).
But yeah, I'll fix it in my code and in the 'New' section.